From 05c5334e3e975e3f88f1c3e40a942f8b6c3fb1aa Mon Sep 17 00:00:00 2001
From: Sta Zhu <zhusidayoyo@hotmail.com>
Date: Sat, 19 Nov 2022 22:51:08 +0800
Subject: [PATCH] Video: Enable HEVC encoding by default

Enable HEVC WebCodec encoding on macOS and Windows by default.
---
 media/base/media_switches.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/media/base/media_switches.cc b/media/base/media_switches.cc
index e9c9c85d0eb0a..c8517b4fcf3e4 100644
--- a/media/base/media_switches.cc
+++ b/media/base/media_switches.cc
@@ -313,7 +313,7 @@ BASE_FEATURE(kPlatformHEVCDecoderSupport,
 // Enables HEVC hardware accelerated encoding for Windows and Mac.
 BASE_FEATURE(kPlatformHEVCEncoderSupport,
              "PlatformHEVCEncoderSupport",
-             base::FEATURE_DISABLED_BY_DEFAULT);
+             base::FEATURE_ENABLED_BY_DEFAULT);
 #endif  // BUILDFLAG(IS_WIN) || BUILDFLAG(IS_MAC)
 #endif  // BUILDFLAG(ENABLE_PLATFORM_HEVC)
 
-- 
2.37.0 (Apple Git-136)

